% Created 2019-03-29 Fri 11:23
% Intended LaTeX compiler: pdflatex
\documentclass[presentation, smaller]{beamer}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{graphicx}
\usepackage{grffile}
\usepackage{longtable}
\usepackage{wrapfig}
\usepackage{rotating}
\usepackage[normalem]{ulem}
\usepackage{amsmath}
\usepackage{textcomp}
\usepackage{amssymb}
\usepackage{capt-of}
\usepackage{hyperref}
\hypersetup{colorlinks,linkcolor=white,urlcolor=blue}
\usepackage{textpos}
\usepackage{textgreek}
\usepackage[version=4]{mhchem}
\usepackage{chemfig}
\usepackage{siunitx}
\usepackage{gensymb}
\usepackage[usenames,dvipsnames]{xcolor}
\usepackage[T1]{fontenc}
\usepackage{lmodern}
\usepackage{verbatim}
\usepackage{tikz}
\usetikzlibrary{shapes.geometric,arrows,decorations.pathmorphing,backgrounds,positioning,fit,petri}
\usetheme{Hannover}
\usecolortheme{whale}
\author{Matthew Henderson, PhD, FCACB}
\date{\today}
\title{Disorders of Oxidative Phosphorylation}
\institute[NSO]{Newborn Screening Ontario | The University of Ottawa}
\titlegraphic{\includegraphics[height=1cm,keepaspectratio]{../logos/NSO_logo.pdf}\includegraphics[height=1cm,keepaspectratio]{../logos/cheo-logo.png} \includegraphics[height=1cm,keepaspectratio]{../logos/UOlogoBW.eps}}
\hypersetup{
 pdfauthor={Matthew Henderson, PhD, FCACB},
 pdftitle={Disorders of Oxidative Phosphorylation},
 pdfkeywords={},
 pdfsubject={},
 pdfcreator={Emacs 26.1 (Org mode 9.1.9)}, 
 pdflang={English}}
\begin{document}

\maketitle

%\logo{\includegraphics[width=1cm,height=1cm,keepaspectratio]{../logos/NSO_logo_small.pdf}~%
%    \includegraphics[width=1cm,height=1cm,keepaspectratio]{../logos/UOlogoBW.eps}%
%}

\vspace{220pt}
\beamertemplatenavigationsymbolsempty
\setbeamertemplate{caption}[numbered]
\setbeamerfont{caption}{size=\tiny}
% \addtobeamertemplate{frametitle}{}{%
% \begin{textblock*}{100mm}(.85\textwidth,-1cm)
% \includegraphics[height=1cm,width=2cm]{cat}
% \end{textblock*}}

\section{Introduction}
\label{sec:orgdb503a6}

\begin{frame}[label={sec:org6b8b4f6}]{ETC and OxPhos system}
\begin{itemize}
\item Responsible for ATP production
\item ETC complexes I-IV
\begin{itemize}
\item There are three energy-transducing enzymes in the electron transport chain - NADH:ubiquinone oxidoreductase (complex I), Coenzyme Q – cytochrome c reductase (complex III), and cytochrome c oxidase (complex IV).[
\end{itemize}
\item OxPhos system complexes I-V
\item IMM proteins
\end{itemize}
\begin{block}{chemiosmotic coupling hypothesis}
\begin{itemize}
\item chemiosmotic coupling hypothesis, proposed by Nobel Prize in Chemistry winner Peter D. Mitchell,
\item the electron transport chain and oxidative phosphorylation are coupled by a proton gradient across the inner mitochondrial membrane.
\item The efflux of protons from the mitochondrial matrix creates an electrochemical gradient (proton gradient).
\item This gradient is used by the F\(_{\text{O}}\)F\(_{\text{1}}\) ATP synthase complex to make ATP via oxidative phosphorylation.
\item ATP synthase is sometimes described as Complex V of the electron transport chain
\end{itemize}

\url{https://upload.wikimedia.org/wikipedia/commons/8/89/Mitochondrial\_electron\_transport\_chain\%E2\%80\%94Etc4.svg}

\url{https://upload.wikimedia.org/wikipedia/commons/1/18/Mitochondrial\_electron\_transport\_chain\_\%28annotated\_diagram\%29.svg}
\end{block}
\end{frame}

\begin{frame}[label={sec:orgddac285}]{Electron Transport Chain}
\begin{itemize}
\item Energy obtained through the transfer of electrons down the ETC is
used to pump protons from the mitochondrial matrix into the
intermembrane space
\begin{itemize}
\item creates an electrochemical proton gradient (\(\Delta\)pH) across the IMM.
\begin{itemize}
\item largely responsible for the mitochondrial membrane potential (\(\Delta \Psi\)M).
\end{itemize}
\item ATP synthase uses flow of \ce{H+} through the enzyme back into the
matrix to generate ATP from ADP and Pi.
\end{itemize}
\item There are three energy-transducing enzymes in the electron transport
chain:
\begin{itemize}
\item NADH:ubiquinone oxidoreductase (complex I)
\item Coenzyme Q – cytochrome c reductase (complex III)
\item cytochrome c oxidase (complex IV).
\end{itemize}
\end{itemize}

\begin{center}
\includegraphics[width=.9\linewidth]{./figures/etc.pdf}
\end{center}
\end{frame}

\begin{frame}[label={sec:orgf17b9b3}]{Electron Transport Chain}
\url{https://www.genome.jp/kegg-bin/show\_pathway?hsa00190}
\end{frame}

\begin{frame}[label={sec:org9dd834e}]{Complex I | NADH-ubiquinone oxidoreductase}
\begin{itemize}
\item catalyzes the transfer of electrons from NADH to coenzyme Q10
(CoQ10) and translocates protons across the inner mitochondrial
membrane

\item Mechanism: 
\begin{enumerate}
\item Seven iron sulfur centers carry electrons from the site of NADH
dehydration to ubiquinone.

\item ubiquinone (CoQ) is reduced to ubiquinol (\ce{CoQH2}).

\item The energy from the redox reaction results in conformational
change allowing hydrogen ions to pass through four transmembrane
helix channels.
\end{enumerate}
\end{itemize}

\begin{center}
\includegraphics[width=.9\linewidth]{./figures/c1.pdf}
\end{center}
\end{frame}


\begin{frame}[label={sec:org5c527a3}]{Complex I | NADH-ubiquinone oxidoreductase}
\url{https://upload.wikimedia.org/wikipedia/commons/4/42/NADH\_Dehydrogenase\_Mechanism\_\%28Fixed\%29.png}
\end{frame}

\begin{frame}[label={sec:org33a5051}]{Complex I | NADH-ubiquinone oxidoreductase Inhibitors}
\begin{itemize}
\item The best-known inhibitor of complex I is rotenone (commonly used as an organic pesticide).
\item Rotenone and rotenoids are isoflavonoids occurring in several genera of tropical plants such as Antonia (Loganiaceae), Derris and Lonchocarpus (Faboideae, Fabaceae).
\item Rotenone binds to the ubiquinone binding site of complex I as well as piericidin A, another potent inhibitor with a close structural homologue to ubiquinone.
\item Bullatacin (an acetogenin found in Asimina triloba fruit) is the most potent known inhibitor of NADH dehydrogenase (ubiquinone) (IC50=1.2 nM, stronger than rotenone).

\item Hydrophobic inhibitors like rotenone or piericidin most likely disrupt the electron transfer between the terminal FeS cluster N2 and ubiquinone.

\item Complex I is also blocked by adenosine diphosphate ribose – a reversible competitive inhibitor of NADH oxidation – by binding to the enzyme at the nucleotide binding site.[39] Both hydrophilic NADH and hydrophobic ubiquinone analogs act at the beginning and the end of the internal electron-transport pathway, respectively.

\item Metformin has been shown to induce a mild and transient inhibition of the mitochondrial respiratory chain complex I, and this inhibition appears to play a key role in its mechanism of action.

\item Inhibition of complex I has been implicated in hepatotoxicity associated with a variety of drugs, for instance flutamide and nefazodone.
\end{itemize}
\end{frame}

\begin{frame}[label={sec:orge4b411c}]{Complex II | Succinate Dehydrogenase}
\begin{itemize}
\item Four subunits compose Complex II of the mitochondrial respiratory chain
\end{itemize}

\begin{center}
\begin{tabular}{ll}
Subunit name & Protein description\\
\hline
SdhA & Succinate dehydrogenase flavoprotein subunit\\
SdhB & Succinate dehydrogenase iron-sulfur subunit\\
SdhC & Succinate dehydrogenase cytochrome b560 subunit\\
SdhD & Succinate dehydrogenase cytochrome b small subunit\\
\end{tabular}
\end{center}

\begin{itemize}
\item The SdhA subunit contains an FAD binding site where succinate
is deprotonated and converted to fumarate.
\end{itemize}

\ce{succinate + ubiquinone ->[CII] fumarate + ubiquinol}

\begin{itemize}
\item Electrons removed from succinate transfer to SdhA
\item transfer across SdhB through iron sulphur clusters to the SdhC/SdhD subunits
\begin{itemize}
\item SdhC/SdhD are anchored in the mitochondrial membrane.
\end{itemize}
\end{itemize}
\end{frame}

\begin{frame}[label={sec:orgaf21b70}]{Complex II | Succinate Dehydrogenase}
\begin{center}
\includegraphics[width=0.9\textwidth]{./figures/SuccDeh.png}
\end{center}
\end{frame}

\begin{frame}[label={sec:orga5979ad}]{Complex III | Coenzyme Q – cytochrome c reductase}
\begin{itemize}
\item Complex III is a multisubunit transmembrane protein encoded by both
the mitochondrial (cytochrome b) and the nuclear genomes (all other
subunits)

\item The bc1 complex contains 11 subunits:
\begin{itemize}
\item 3 respiratory subunits (cytochrome B, cytochrome C1, Rieske protein)
\item 2 core proteins
\item 6 low-molecular weight proteins
\end{itemize}
\end{itemize}

QH\(_{\text{2}}\) + 2 cytochrome c (Fe\(^{\text{3+}}\)) + 2 H\(^{\text{+}}_{\text{in}}\) \(\to\)  Q + 2 cytochrome c (Fe\(^{\text{2+}}\)) + 4 H\(^{\text{+}}_{\text{out}}\)
\end{frame}


\begin{frame}[label={sec:org7ac3abd}]{Complex III | Coenzyme Q – cytochrome c reductase}
\url{https://upload.wikimedia.org/wikipedia/commons/e/e2/Complex\_III.png}
\end{frame}

\begin{frame}[label={sec:org3d281d8}]{Complex III | Coenzyme Q – cytochrome c reductase Inhibitors}
\begin{itemize}
\item There are three distinct groups of Complex III inhibitors:
\begin{itemize}
\item Antimycin A binds to the Q\(_{\text{i}}\) site and inhibits the transfer of electrons in Complex III from heme b\(_{\text{H}}\) to oxidized Q (Q\(_{\text{i}}\) site inhibitor).
\item Myxothiazol and stigmatellin binds to the Q\(_{\text{o}}\) site and inhibits the transfer of electrons from reduced QH\(_{\text{2}}\) to the Rieske Iron sulfur protein.
\begin{itemize}
\item Myxothiazol and stigmatellin bind to distinct but overlapping pockets within the Q\(_{\text{o}}\) site.
\item Myxothiazol binds nearer to cytochrome bL (hence termed a "proximal" inhibitor).
\item Stigmatellin binds farther from heme bL and nearer the Rieske Iron sulfur protein, with which it strongly interacts.
\end{itemize}
\end{itemize}
\end{itemize}
\end{frame}

\begin{frame}[label={sec:orgb5a50c4}]{Complex IV | Cytochrome c oxidase}
\begin{itemize}
\item last enzyme in the respiratory electron transport chain.
\item large IMM integral membrane protein composed of several metal prosthetic sites and 14 protein subunits.
\item eleven subunits are nuclear in origin, and three are synthesized in the mitochondria. 
\begin{itemize}
\item contains two hemes,
\item cytochrome a and cytochrome a3,
\item two copper centers, CuA and CuB
\end{itemize}
\item the cytochrome a3 and CuB form a binuclear center that is the site of oxygen reduction.

\item receives an electron from four cytochrome c molecules and transfers them to one dioxygen molecule
\begin{itemize}
\item converting the molecular oxygen to two molecules of water.
\item In the process binds four protons from the inner aqueous phase
to make two water molecules, and translocates another four protons
across the membrane, increasing the transmembrane difference of
proton electrochemical potential which the ATP synthase then uses to
synthesize ATP.
\end{itemize}
\end{itemize}

4 Fe\(^{\text{2+}}\)-cytochrome c + 8 H\(^{\text{+}}_{\text{in}}\) + O\(_{\text{2}}\) \(\to\)  4 Fe\(^{\text{3+}}\)-cytochrome c + 2H\(_{\text{2}}\)O + 4 H\(^{\text{+}}_{\text{out}}\)
\end{frame}


\begin{frame}[label={sec:org03f1af2}]{Complex IV | Cytochrome c oxidase Inhibitors}
\begin{itemize}
\item Cyanide, azide, and carbon monoxide all bind to cytochrome c
oxidase

\item Nitric oxide and hydrogen sulfide, can also inhibit COX by
binding to regulatory sites on the enzyme
\end{itemize}
\end{frame}

\begin{frame}[label={sec:orgd9b9eee}]{Complex V | ATP synthase}
\begin{itemize}
\item formation of ATP from ADP and Pi is energetically unfavorable
\item ATP synthase couples ATP synthesis to an electrochemical gradient (\(\Delta \Psi\)M).
\end{itemize}

\begin{center}
\includegraphics[width=0.5\textwidth]{./figures/atp_synthase.jpg}
\label{org2265d32}
\end{center}

\centering
\ce{ADP + Pi + H+_{out} <=> ATP + H2O + H+_{in}}
\end{frame}
\end{document}